src/data/validators/validators.ts

```diff
@@ -0,0 +1,29 @@
+const PASSWORD_REGEX = /^(?=.*\d)(?=.*[a-zA-Z]).+$/;
+const EMAIL_REGEX = /^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/;
+
+export function passwordValidator(password: string) {
+  const minLength = 6;
+  const maxLength = 30;
+
+  if (password.length < minLength) {
+    throw new Error('A valid password should have at least 6 characters.');
+  }
+
+  if (password.length > maxLength) {
+    throw new Error('Your password surpasses the limit of 30 characters.');
```

Comment:
se o max e o min variarem, vc pode usar a constante na mensagem, para nao precisar editar 2x

```suggestion
    throw new Error(`A valid password should have at least ${minLenght} characters.`);
  }

  if (password.length > maxLength) {
    throw new Error(`Your password surpasses the limit of ${maxLenght} characters.`);
```